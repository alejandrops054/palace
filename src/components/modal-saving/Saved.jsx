/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from "react";
import '../../css/ui/spinner.css';

const Saved = ({ title = "Changes saved succesfully" }) => {
  return (
    <div className="container mx-auto">
      <i className="fa fa-check-circle fa-5x"></i>
      <h6 className="mt-4 text-white">{title}</h6>
    </div>
  );
};

export default Saved;