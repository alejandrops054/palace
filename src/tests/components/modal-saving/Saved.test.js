/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import Saved from '../../../components/modal-saving/Saved'

describe('Saved component',() => {
    test('Debe mostrar snapshot', () => {
        
        const wrapper = shallow(<Saved/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})