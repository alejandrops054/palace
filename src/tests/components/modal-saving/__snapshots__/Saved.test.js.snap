// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Saved component Debe mostrar snapshot 1`] = `
<div
  className="container mx-auto"
>
  <i
    className="fa fa-check-circle fa-5x"
  />
  <h6
    className="mt-4 text-white"
  >
    Changes saved succesfully
  </h6>
</div>
`;
